client-app/ui-kit/composables/useQuantityValidationSchema.ts

```diff
@@ -3,32 +3,65 @@ import { useI18n } from "vue-i18n";
 import { number } from "yup";
 import type { NumberSchema } from "yup";
 
-export function useQuantityValidationSchema(minQuantity?: number, maxQuantity?: number) {
+export function useQuantityValidationSchema(payload: {
```

Comment:
I think it's better to make `payload` fields `Ref` or `MayBeRef`, so schema will be reactive and you will not need to use `.value` when pass variables here

Reply:
TBD

Reply:
Suggest to refactor composable signature later